Workbooks/ADXCluster/AtResource/ResourceInsights.workbook

```diff
@@ -897,171 +3619,5523 @@
+                          "Subscription"
+                        ],
+                        "expandTopLevel": true,
+                        "finalBy": "Name"
+                      },
+                      "labelSettings": [
+                        {
+                          "columnId": "Subscription"
+                        },
+                        {
+                          "columnId": "Name"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-CacheUtilization",
+                          "label": "Cache utilization (Average)"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-CacheUtilization Timeline",
+                          "label": "Cache utilization Timeline"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-CPU",
+                          "label": "CPU (Average)"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-CPU Timeline",
+                          "label": "CPU Timeline"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-IngestionUtilization",
+                          "label": "Ingestion utilization (Average)"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-IngestionUtilization Timeline",
+                          "label": "Ingestion utilization Timeline"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-InstanceCount",
+                          "label": "Instance Count (Average)"
+                        },
+                        {
+                          "columnId": "microsoft.kusto/clusters-Cluster health-InstanceCount Timeline",
+                          "label": "Instance Count (Average) Timeline"
+                        }
+                      ]
+                    },
+                    "sortBy": [],
+                    "showExportToExcel": true
+                  },
+                  "customWidth": "25",
+                  "showPin": true,
+                  "name": "Streaming Ingest Result"
+                }
+              ]
+            },
+            "name": "Streaming Ingestion",
+            "styleSettings": {
+              "showBorder": true
+            }
           }
         ]
       },
       "conditionalVisibility": {
         "parameterName": "tab",
         "comparison": "isEqualTo",
-        "value": "queryperformance"
+        "value": "Cluster"
       },
-      "name": "Query Performance"
+      "name": "Cluster Group"
     }
   ],
+  "fallbackResourceIds": [
```

Comment:
we can delete this `fallbackResouceIds` object. For templates we don't want to have any specific sub/rg information on them